content/influxdb/v2.0/reference/cli/influx/org/members/add.md

```diff
@@ -16,15 +16,15 @@ influx org members add [flags]
 ```
```

Comment:
I notice these reference files are still stored in a `v2.0` folder, should they be copied to a `v2.1` and edited there?

Reply:
@danxmoran The minor version directories right now are scoped to OSS releases. As far as we understand, the new CLI is being released immediately with a 2.1.0 tag, but the full OSS release won't happen until the end of Augusts. So for now, we're making all the CLI-specific changes in the v2.0 directory to support the new CLI release and will port everything over to 2.1 as we get closer to the full 2.1 release.